sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -85,3 +85,23 @@
 --    arguments: none
 --    behavior: can take a long time.
 
+
+
+#IfMissingColumn list_options edit_options
+  ALTER TABLE `list_options` ADD `edit_options` TINYINT(1) NOT NULL DEFAULT '1';
+#Endif
+
+#IfNotTable zf_multiple_db
```

Comment:
that keeps this generic and call it multiple_db
(since it may get expanded to use outside of zend in the future)